source/main/utils/getRuntimeFolderPath.js

```diff
@@ -1,6 +1,12 @@
 import path from 'path';
 
+const isProd = process.env.NODE_ENV === 'production';
+
 export default (platform, env, appName) => {
+  if (!isProd) {
+    return './';
```

Comment:
@disassembler shouldn't we use `path.join` here?

Reply:
what would be combining, since it's just the current directory?

Reply:
Ah yes... Sorry for my dumb question... All good :)